blog: how to do sales with no experience (#9503)
PostHog/posthog.com: contents/blog/sales-no-experience.md

---
date: 2024-09-30
title: How to do sales with no experience
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - james-hawkins
---

"I just need to hire a sales person so we get some revenue". There it is people, the start of the end.

Here's the rest of the movie:

* salesperson starts
* they get a few meetings, they're pretty random feeling
* you start building stuff that people in the meetings have talked about
* nothing happens for ages, or at all

> * you now don't know if the salesperson is at fault, or if you don't have product market fit

Reviewer: This paragraph later:

> Everything else is a distraction early on. So what are you doing if you don't care about this? Likely being avoidant from all the rejection and fear of failure! You may not want to hear it but a magical salesperson will be _far_ less likely to figure this out than you.

Especially the "magical salesperson" bit feels like a clearer hook. Something like telling the "The myth of the magical sales person" of that makes sense.

## Who I'm writing this for

> Are you the founder of a startup that has never sold before, and you're trying to figure out product market fit? This is written for you.

Reviewer: I think we can leave this implied by combining with the next section

## One job, people

You have one job. Get to product market fit.

> It's a _terrible_ idea to out

Author: ```suggestion
It's a _terrible_ idea to outsource sales to someone else. It's half the battle.
```

It _could_ of course work, but it just adds another variable to the most important and most multivariate problem you will _ever_ solve as a startup.

## A little pep talk

_You_ are the best salesperson for your product. There will _never_ be anyone else as well qualified to sell.

* You get the _why_ behind what you're building
* You are the parent of your startup (TODO INSERT LINK TO AIRBNB FOUNDER MODE BIT ON THIS). This means you are able to tweak things - like shipping an extra feature on the spot, your pricing, or how you pitch the product.

You need to prioritize figuring out making money as one of the very top things:

* do users like your product
* do users pay for it

Everything else is a distraction early on. So what are you doing if you don't care about this? Likely being avoidant from all the rejection and fear of failure! You may not want to hear it but a magical salesperson will be _far_ less likely to figure this out than you.

> ## Getting meetings

Reviewer: I feel like there should be a tiny transition from:

> * do users like your product
> * do users pay for it

To  "getting meetings", something like "at the beginning, the only way to get people to pay for your product is to ask them, you need to talk to them, and that requires meetings"

Things that worked to recruit users:

* Friends
* People you've worked with before
* Posting on your social media
* LinkedIn DMs - customized
* Cold emails - customized
* Posting our startup to places where our users hang out (like HackerNews for PostHog), in a non spammy way

Things that didn't really work:

* Going to events - this just felt awkward and spammy
* Automated emails or DMs - you aren't trying to scale up, you are trying to get just a few paying customers early on, so don't try to scale up your outreach, focus on the quality early on

## Running a sales meeting

As a more technical person, the first thing to be clear on is that this is a _sales_ meeting not a _product_ meeting.

In _both_ types of meeting, you should be asking lots of questions. However, there are some key differences:

* In sales meetings, you are aiming to _close down_ objections. In product meetings you want to _expand_ on them.
* In sales meetings, you are steering the customer. In product meetings, you should let users meander.

> * In sales meetings, you learn by seeing if it closes or not. In product meetings, you learn based on what people are saying.

Reviewer: I like this bit :)

> We should talk vibes for a moment. First, don't make it painful for the customer to interact with you. _You_ are leading the meeting because you asked forit. A good way of doing this is to have some friendly rapport at the beginning, then layout "hey so there are four things I hoped to run through with you in this meeting. First, I want to get a grip on your company and priorities, second I'd like to cover X problem space a bit, and third, I wanted to talk about our product and show you a few relevant bits of it. Is that a good use of the next 25 minutes from your perspective?" Asking for _permission_ to ask questions means it'll feel less awkward when you do. Because you should.

Reviewer: ```suggestion
We should talk vibes for a moment. First, don't make it painful for the customer to interact with you. _You_ are leading the meeting because you asked for it. A good way of doing this is to have some friendly rapport at the beginning, then layout "hey so there are four things I hoped to run through with you in this meeting. First, I want to get a grip on your company and priorities, second I'd like to cover X problem space a bit, and third, I wanted to talk about our product and show you a few relevant bits of it. Is that a good use of the next 25 minutes from your perspective?" Asking for _permission_ to ask questions means it'll feel less awkward when you do. Because you should.
```

* "I am willing to bet that you have an issue with X in your org, but tell me how much of a priority it is"
* "Something I've seen from most of my other customer conversations is they find X painful"

Be proud of who you are. When you're tiny, it can feel like you have no hope - you're maybe one or two people up against unicorn companies. Why would _anyone_ pick your product that _even you_ don't have complete confidence in yet? Well, there are many good reasons. First, it's very cool getting to meet the person that built the actual product. That's not an experience that customers ever normally get. This makes _you_ an interesting person to talk to. You can explain authentically _why_ you thought your product was worth building, and why you're excited about it. Be honest about what you do and do not know may be most useful. If you've never had a single call before, own that - "hey, I'm building this thing, you are _literally the first person_ I've ever talked to about it, so would you mind going gently with me?" Don't lie or pretend to be a fortune 500 if you're just one person! Customers are as clever as you and can see through it.

## Closing a deal

> * Work backwards. Ask the customer something like "so to get you up and running as a customer, what are all the steps needed?" then get them to go into detail. Do they actually _have_ the budget or do they need to get it? If they need to get it, who do they get it from? Could you run a demo for the other person?

Author: ```suggestion
Start by working backwards. Ask the customer something like "so to get you up and running as a customer, what are all the steps needed?" then get them to go into detail. Do they actually _have_ the budget or do they need to get it? If they need to get it, who do they get it from? Could you run a demo for the other person?
```
